src/utils/helpers/with-unique-name/with-unique-name.hoc.js

```diff
@@ -24,8 +24,8 @@ const withUniqueName = (WrappedComponent) => {
       );
     }
   }
-  const displayName = WrappedComponent.displayName || WrappedComponent.name || 'Component';
-  WithUniqueName.displayName = `WithUniqueName(${displayName})`;
+  const displayName = WrappedComponent.displayName || WrappedComponent.name || 'WithUniqueName(Unknown)';
+  WithUniqueName.displayName = displayName.replace('Base', '');
```

Comment:
Same suggestion as above:
```suggestion
  WithUniqueName.displayName = displayName.replace(/^Base/, '');
```

Reply:
Fixed in 2f027d8

Reply:
Thanks :thumbsup: